### DISCUSSION TITLE
set 'encrypt' option to true [Jan 14]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#4904] [Discord Threads]
> 🙋‍♂️ **ashtontan**  
## Need help on below error message, try to connect nocodb using external database Azure Microsoft SQL  
![image0](https://cdn.discordapp.com/attachments/661905455894888493/1052883105943392266/image.png)

**Comment by o1lab** [comment]
> 🙋‍♂️ **wkw**  
#### you may try following   
#### - click `SSL & Advanded Parameters`   
#### - add the option in `Extra connection parameters`  
![image0](https://cdn.discordapp.com/attachments/1052883106308313188/1052885003874353262/image.png)  
#  
> 🙋‍♂️ **ashtontan**  
#### <@495466997304262658>, do u have any documentation on this or can u show me the parameters, thank  
#  
> 🙋‍♂️ **wkw**  
#### you may check the azure docs. based on the error message, you may try    
####    
#### ```   
#### "encrypt": "true"   
#### ```  
#  
> 💁 **o1lab**  
#### we use knex underneath, so if u can make it work for knex. you can even edit the connection json. Its within ssl & advanced params  
#  
> 🙋‍♂️ **ashtontan**  
#### <@495466997304262658>, try set base on advise still have error message, try .T. also the same error  
![image0](https://cdn.discordapp.com/attachments/1052883106308313188/1052983834733457441/image.png)  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1052883106308313188)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
